sklearn/feature_extraction/text.py

```diff
@@ -1062,6 +1062,12 @@ class TfidfTransformer(BaseEstimator, TransformerMixin):
     sublinear_tf : boolean, default=False
         Apply sublinear tf scaling, i.e. replace tf with 1 + log(tf).
 
+    Attributes
+    ----------
+    idf_ : array, shape = [n_features], or None
```

Comment:
also here,
```
idf_ : array, shape [n_features]
```

Reply:
Right. In fact, this was a copy and paste from TfidfVectorizer documentation, which is incorrect today. I changed it in both places to not defined.